spiceaidocs/docs/features/caching/index.md

```diff
@@ -21,14 +21,16 @@ name: app
 runtime:
   results_cache:
     enabled: true
-    cache_max_size: 128mb
-    item_expire: 1s
+    cache_max_size: 128MiB
+    eviction_policy: lru 
+    item_ttl: 1s
 ```
 
-- `enabled` - optional, `true` by default (if there is a non-empty `results_cache` section defined)
-- `cache_max_size` - optional, maximum cache size. Default is `128MB`
-- `item_expire` - optional, cache entry expiration time, 1 second by default.
+- `enabled` - optional, `true` by default
+- `cache_max_size` - optional, maximum cache size. Default is `128MiB`
+- `eviction_policy` - optional, cache replacement policy when the cached data reaches the `cache_max_size`. Default is `lru` - [least-recently-used (LRU)](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU)
+- `item_ttl` - optional, cache entry expiration duration, 1 second by default.
 
 :::warning[Limitations]
-- Results caching for Arrow Flight queries is not currently supported (coming soon).
+- Cache currently supports only the [least-recently-used (LRU)](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU) replacement policy.
```

Comment:
I don't think this should be a warning/limitation. It's just how it works.

Reply:
Updated (moved to `eviction_policy` description): https://github.com/spiceai/docs/pull/257/commits/640af4b05d9e40775eb710d707ffcef37dcc06bb